### ISSUE TITLE
Dashboards (timerange): Open multiple-timeranges in a new tabs

### ISSUE THREAD

**Issue by jankaifer** [grafana/grafana#93048]
> I already have this implemented and I'd like to upstream my changes. I opened this issue to discuss whether it is suitable for upstreaming.
> I want to get explicit confirmation that this contribution is welcome before spending time on cleaning up the code and separating it into a separate PR.

**Why is this needed**:

When investigating something there might be multiple situations that are worth digging into.
Currently I need to:
1. Zoom-in to 1. situation
2. Wait for it to load
3. Go back
4. Wait for it to load
5. Zoom in to 2. situation
6. Wait for it to load
7. ...

And Imagine you are doing this for 100+ different situations each day.

**What would you like to be added**:

What I currently do using my feature is:
1. Press alt
2. Select situation 1.
3. Select situation 2.
4. Select situation 3.
5. Release alt
6. All situations open in new tabs and they start loading in parallel
7. Check situation 1.
8. Check situation 2.
9. Check situation 3.

This is significantly faster than checking them one by one in a single tab.

**Who is this feature for?**

I use Grafana to investigate financial markets, but this will benefit all power users using Grafana as their main interface for data exploration.
